src/app/pipes/number-transform/number-transform.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'appNumberTransform'
+})
+export class NumberTransformPipe implements PipeTransform {
+
```

Comment:
Prettier doesn't allow these blank spaces. Please don't forget to use it. You have to install it and run the keys combination according to your operating system.